packages/magner/lib/types/form/base.ts

```diff
@@ -37,6 +43,9 @@ export interface BaseProps {
 
   /** If `readOnly: true`, parse the value of the form field to display as a string */
   readOnlyFormatter?: (val: unknown) => string,
+
+  /** We keep pure functions in this object */
+  inner: InnerConditions
```

Comment:
Suggestion: change the comment to `Magner's inner properties. No need to change`. Right now the comment is somehow misleading